app.rb

```diff
@@ -0,0 +1,70 @@
+require 'sinatra/base'
+require_relative 'models/user'
+require_relative 'models/peep'
+require_relative 'models/datamapper_setup'
+
+ENV['RACK_ENV'] ||= 'development'
+
+DataMapper.finalize
+DataMapper.auto_upgrade!
+
+class Chitter < Sinatra::Base
+
+  enable :sessions
+
+  get '/' do
+    redirect '/home'
+  end
+
+  get '/sign-up' do
+    erb :sign_up
+  end
+
+  post '/new-user' do
+    User.new_user_create(params[:name], params[:username], params[:email], params[:password])
+    session["username"] = params[:username] if User.redirect == '/home'
+    redirect User.redirect
```

Comment:
`User.redirect` is a cool idea.  It may cause errors if you have multiple people clicking around your site at the same time, because the `redirect` instance variable will be shared between them.